clangd/index/ClangdIndexerImpl.cpp

```diff
@@ -107,15 +136,16 @@ class TranslationUnitToBeIndexedCollector {
                 Collector(Collector) {
           }
           virtual ClangdIndexFile::NodeVisitResult VisitDependent(ClangdIndexFile& File) override {
-            StringRef Path = File.getPath();
-            llvm::errs() << " Considering dependent : " << Path << "\n";
-            llvm::sys::fs::directory_entry Entry(Path);
+            StringRef FilePath = File.getPath();
+            llvm::errs() << " Considering dependent : " << FilePath << "\n";
+            llvm::sys::fs::directory_entry Entry(FilePath);
             auto Status = Entry.status();
             if (!Status) {
               llvm::errs() << " Cannot stat file. Nothing to do. " << "\n";
               return ClangdIndexFile::NodeVisitResult::CONTINUE;
             }
-            Collector.visitFile(Entry, false);
+            std::vector<Path> EmptyList;
```

Comment:
we shouldn't pass an empty list here. We should still use exlusions I think because otherwise it would be possible to be indexing dependent files that are in the index but are now excluded after the user has changed the config. It does raise the question of whether or not we should delete files from the index when exclusions change. Perhaps it's not worth it, let's not worry about this part for now.